extensions/vscode/tsconfig.json

```diff
@@ -1,24 +1,17 @@
 {
-    "compilerOptions": {
-        "lib": [
-            "ES2019"
-        ],
-        "module": "commonjs",
-        "outDir": "out",
-        "rootDir": "src",
-        "sourceMap": true,
-        "target": "es2019"
-    },
-    "exclude": [
-        "node_modules",
-        ".vscode-test"
-    ],
-    "include": [
-        "src"
-    ],
-    "references": [
-        {
-            "path": "./lsp-client"
-        }
-    ]
+	"compilerOptions": {
+		"module": "Node16",
+		"target": "ES2022",
+		"outDir": "out",
+		"lib": [
+			"ES2022"
+		],
+		"sourceMap": true,
+		"rootDir": "src",
+		"strict": true   /* enable all strict type-checking options */
+		/* Additional Checks */
+		// "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
+		// "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
+		// "noUnusedParameters": true,  /* Report errors on unused parameters. */
```

Comment:
Do we fail some of these additional checks? I think we should either turn them on or delete this comment.

Reply:
not that I know of no, once again generated code. I can go mess with them